### ISSUE TITLE
Running the remote debugger

### ISSUE THREAD

**Issue by fakedrake** [metaskills/mocha-phantomjs#156]
Is there a way to run the [remote debugger](http://phantomjs.org/troubleshooting.html) from mocha-phantomjs?

**Comment by nathanboktae** [comment]
```
phantomjs --remote-debugger-port=9000 lib/mocha-phantomjs.coffee <your tests> <reporter> <config as json>
```

**Comment by nathanboktae** [comment]
Tracking the long term fix for this in #186
